docs/paper/admin/reference/built-in-commands.md

```diff
@@ -0,0 +1,187 @@
+---
+slug: /reference/built-in-commands
+---
+
+
+# Built-In Commands
+
+Paper includes a few commands by default.
+
+## The `/paper` command
+Permission: `bukkit.command.paper`
+
+It contains many subcommands.
+
+
+### `/paper debug`
+Permission: `bukkit.command.paper.debug` <br>
+Usage: `/paper debug chunks`
+
+When executed, a file with info for the server will be generated in the `debug` folder.
+
+
+### `/paper mobcaps`
+Permission: `bukkit.command.paper.mobcaps` <br>
+Usage: `/paper mobcaps [world]` Replace [world] with a specific world name.
+
+Not done
+
+
+### `/paper syncloadinfo`
+Permission: `bukkit.command.paper.syncloadinfo`
+:::note
+This command requires the `-Dpaper.debug-sync-loads=true` startup flag.
```

Comment:
Link to the [docs page](https://docs.papermc.io/paper/reference/system-properties#paperdebug-sync-loads) about this system property